services/reddit/subreddit-subscribers.service.js

```diff
@@ -0,0 +1,70 @@
+'use strict'
+
+const { BaseJsonService } = require('..')
+const Joi = require('joi')
+const { nonNegativeInteger } = require('../validators')
+const { metric } = require('../text-formatters')
+
+const schema = Joi.object({
+  data: Joi.object({
+    subscribers: nonNegativeInteger,
```

Comment:
Let's try making this optional to account for the way the API response is representing not found scenarios.
```suggestion
    subscribers: optionalNonNegativeInteger,
```